subt/main.py

```diff
@@ -118,6 +118,7 @@ def __init__(self, config, bus):
             self.virtual_bumper = VirtualBumper(timedelta(seconds=virtual_bumper_sec), virtual_bumper_radius)
         self.speed_policy = config.get('speed_policy', 'always_forward')
         assert(self.speed_policy in ['always_forward', 'conservative'])
+        self.height_above_ground = config.get('height_above_ground', 0.0)
```

Comment:
This is surprising. Overall, it is ignored, right?

Reply:
It become part of the "enter the area" trajectory. The reason is that we measure distance in 3d when following the waypoints. So when a drone flies high or we overestimate the Z axis, we never reach the waypoint at 0.0 altitude and keep circling around.